doc/releases/release-notes-3.4.rst

```diff
@@ -489,7 +489,7 @@ Bluetooth
     ``ticks_slot_window``. Implement continuous scanning with it.
   * Added support for considering the SDU interval, along with the packet
     sequence number and time stamps, in SDU fragmentation.
-  * Added a new ``BT_CTRL_TX_PWR_DBM`` option to set the TX power directly in
+  * Added a new ``BT_CTLR_TX_PWR_DBM`` option to set the TX power directly in
```

Comment:
```suggestion
  * Added a new ``BT_CTRL_TX_PWR_DBM`` option to set the TX power directly in
```

Old release notes should not be modified

Reply:
@kartben need you inputs here on fixing typo in old release notes.

Reply:
@kartben you have always made changes to old release notes after releases... hope you can provide your inputs to the typo fix I have done in this PR...

Reply:
I would not necessarily make this change. It's true that past release notes are sometimes touched, but usually only to align with doc infrastructure changes

Reply:
> usually only to align with doc infrastructure changes

Is this true and documented as the process? I am inspired by git history, many changes post release is done to release notes... one such recent change: 

https://github.com/zephyrproject-rtos/zephyr/commit/a43009079f6670ef1085603b0474d147ffdb76f4#diff-f981d3b84e83c3bfe2309beadeac87f52c8c9e99fe715f96813d87ecc87acd8eR417-R422

May be i can add to documentation that release note shall not be updated post release, if you point me to a right section/file...